src/hooks/useDailyXvs.ts

```diff
@@ -0,0 +1,36 @@
+import { useMemo } from 'react';
+import { BigNumber } from 'bignumber.js';
+import { useGetVTokenDailyXvs } from 'clients/api';
+import { convertWeiToCoins } from 'utilities/common';
+import { Asset } from 'types';
+import { XVS_TOKEN_ID } from 'constants/xvs';
+
+export const useDailyXvs = ({
+  accountAddress,
+  assets,
+}: {
+  accountAddress?: string;
+  assets: Asset[];
+}) => {
```

Comment:
Passing the account address is a good call, however I think `assets` should be grabbed from the hook directly using `useGetAssets`. This way this hook can be used from anywhere without needing to receive anything else than the account address.
What do you think?

Reply:
nice point

Reply:
do you mean `useGetAssetsInAccount` here?

Reply:
removed props for hook, used `getMarketsData` for `xvsPriceDollars`